### PR TITLE
feat(bump): mversion for bumping

### PR DESCRIPTION
**Pull request by SyntaxStacks**
https://jira.rax.io/browse/RSWEB-4695

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,10 +6,9 @@
     "Brady Vitrano <brady.vitrano@rackspace.com>"
   ],
   "scripts": {
-    "test": "npm run grunt test",
-    "build": "npm run grunt build",
-    "bower": "./node_modules/bower/bin/bower ",
-    "grunt": "./node_modules/grunt-cli/bin/grunt "
+    "test": "grunt test",
+    "build": "grunt build",
+    "bower": "./node_modules/.bin/bower "
```
Comment (bighappyface) [inline package.json]: @SyntaxStacks we can just use `bower ` here

**Comment by bighappyface** [comment]
@SyntaxStacks will you add an `mversion` npm script and put some instructions in the readme?

**Comment by amberRrucker** [comment]
+1. Looks fine to me.

**Comment by bighappyface** [comment]
+1

**Pull request merged**